pyproject.toml

```diff
@@ -25,6 +25,8 @@ qibo = "^0.2.11"
 qibo_client = "^0.0.9"
 qiskit_ibm_runtime = ">=0.17"
 qiskit_ibm_provider = ">=0.8.0"
+qiskit_braket_provider = { version = "^0.3.1", optional = true }
+amazon-braket-sdk = { version = "^1.83.0", optional = true }
```

Comment:
I would also add them as mandatory to the `test.dependencies` below, this way you don't have to change the poetry command in the workflow above.

Reply:
Done

Reply:
Thanks so much @TL231!